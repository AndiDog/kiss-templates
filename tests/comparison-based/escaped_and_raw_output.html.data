struct
{
  std::string myhref = "<test':_:'me\">";
} data;
