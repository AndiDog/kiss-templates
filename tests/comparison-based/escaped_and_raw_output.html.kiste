%namespace comparison_based_test
%{
  $class EscapedAndRawOutput

  %auto render() -> void
  %{
<a href="${data.myhref}" $raw{"class=\"mycssclass\""}>Link</a>
  %}

  $endclass
%}
