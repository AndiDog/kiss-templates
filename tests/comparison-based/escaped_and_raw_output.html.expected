<a href="&lt;test&#39;:_:&#39;me&quot;&gt;" class="mycssclass">Link</a>
